polaris-react/src/styles/_common.scss

```
@@ -17,3 +17,5 @@
 @import './shared/interaction-state';
 
 @import '../../../node_modules/@shopify/polaris-tokens/dist/scss/media-queries';
+
+$se23: "[class='Polaris-summer-editions-2023']";
```

minor suggestion/nit:
```suggestion
$se23: "[class='Polaris-Summer-Editions-2023']";
```

---

I actually had summer capitalized at first hah, not really sure why I switched it